game-core/src/main/java/games/strategy/triplea/attachments/UnitAttachment.java

```diff
@@ -3267,9 +3267,9 @@ public String toStringShortAndOnlyImportantDifferences(final GamePlayer player)
                 + (support.getAllied() && support.getEnemy()
                     ? " Allied & Enemy "
                     : (support.getAllied() ? " Allied " : " Enemy "))
-                + (support.getUnitType().size() > 4
+                + (support.getUnitType().size() > 25
                     ? "Units"
-                    : MyFormatter.defaultNamedToTextList(support.getUnitType(), "/", false));
+                    : MyFormatter.defaultNamedToTextList(support.getUnitType()));
```

Comment:
Change unit delimiter to comma instead of slash as it works better with line breaks.